packages/ButtonLink/ButtonLink.jsx

```diff
@@ -43,6 +43,14 @@ const StyledButtonLink = styled(StyledButton)(
         color: hoverColor,
       },
     }
+  },
+  ({ fullWidth }) => {
+    const width = fullWidth === 'true' ? '100%' : 'auto'
```

Comment:
This line should be changed to `const width = fullWidth ? '100%' : 'auto'` as `fullWidth` should no longer be a string.

Reply:
Hi @jraff,
I have update your requested change.